shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/model/FeedingPointModel.kt

```diff
@@ -19,7 +19,8 @@ data class FeedingPointModel(
     val isFavourite: Boolean = false,
     val lastFeeder: Feeder,
     val remoteness: Remoteness = Remoteness.ANY,
-    val coordinates: Point
+    val coordinates: Point,
+    val images: List<String> = emptyList()
```

Comment:
Let's keep this model simple and store not a list of images, but only a first image. 
You pass only the first one anyway

Reply:
this model only? I was thinking the whole implementation grab first element only and I'm not sure if there is a scenario where we use the whole images list.